pages/mintKamon/index.tsx

```diff
@@ -55,67 +55,74 @@ const MintKamon: NextPage = () => {
   const { totalSupply } = useTotalSupply()
 
   useEffect(() => {
-    setTokenURI('httsp://yourtokenURI') // TODO: for enable and mintWithHenkaku method
     setHasNFT(balanceOf && Number(balanceOf.toString()) > 0 ? true : false)
   }, [])
 
-  if (false && !tokenURI) {
-    return <GenerateImageForm />
+  if (mounted && data?.address && !tokenURI) {
```

Comment:
```suggestion
  if (mounted && data?.address && !hasNFT && !tokenURI) {
```

This is also a bit of aggressive